packages/notifier/README.md

```diff
@@ -94,55 +102,54 @@ observeIteration(subscription, observer);
-AsyncIterable, a remote presence of an AsyncIterable, or a local or remote
-promise for an AsyncIterable. `observeIteration` only sends it eventual
-messages using `E` (equivalent to the tildot syntax `~.`), and so doesn't care
-about these differences.
-
-Although Bob's code is correct, it is sub-optimal. Its distributed systems
-properties are not terrible, but Bob can do better using the
-`getSharableSubsciptionInternals()` method provided by both NotifierKit and
-SubscriptionKit. This enables consumer Bob to make a local AsyncIterable that
-coordinates better with producer Paula's IterationObserver. Alice's code is
-less forgiving. She's using JavaScript's `for-await-of` loop which requires a
-local AsyncIterable. It cannot handle a remote reference to an AsyncIterable
-at
-Paula's site. Alice has no choice but to make an AsyncIterable at her site.
-Using `getSharableSubsciptionInternals()` is the best way for her to do so. She
-can replace her last line, the call to `consume(subscription)` with
+Either makeNotifierKit or makeSubscriberKit can be used in a multicast manner with good 
+distributed systems properties, where there is only one producing site but any number of 
+consuming sites. The producer is not vulnerable to the consumers; they cannot cause the kit 
+to malfunction or prevent the code producing values from making progress. The consumers are 
+not vulnerable to each other; one can’t cause other consumers to hang or miss values.
+
+For distributed operation, all the iteration values---non-final values, successful completion 
+value, failure reason---must be `Passable`; values that can somehow be passed between vats. 
+The rest of this doc assumes all these values are Passable.
+
+The makeNotifierKit() or makeSubscriptionKit() call makes the notifier/updater or 
+publication/subscription pair on the producer's site. As a result, both the `iterationObserver` and 
+the initial `asyncIterable` are on the producer's site. If Producer Paula sends Consumer Bob 
+the `asyncIterable`, Bob receives a possibly remote reference to the asyncIterable. Consumers can 
+be remote from the producer of their consumed content. 
+
+Bob's code above is still correct if he uses this reference directly, since `observeIteration` only
+needs its first argument to be a reference of some sort to an AsyncIterable conveying Passable 
+values. This reference may be a local AsyncIterable, a remote presence of an AsyncIterable, or a 
+local or remote promise for an AsyncIterable. `observeIteration` only sends it eventual messages 
+using `E` (equivalent to the tildot syntax `~.`), and so doesn't care about these differences.
+
+While correct, Bob’s code is sub-optimal. Its distributed systems properties are not terrible, but 
+Bob does better using `getSharableSubscriptionInternals()` (provided by both NotifierKit and 
+SubscriptionKit). This lets Bob make a local AsyncIterable that coordinates better with producer 
```

Comment:
My mistake. I changed the design and only partially updated the doc, leaving it in an inconsistent state. The method for the subscription kit is `getSharableSubscriptionInternals`, but the method for the notifier kit is `getSharableNotifierInternals`. Where we revisit it below the text is correct.